lang/rust/stone.yml

```diff
@@ -0,0 +1,42 @@
+#
+# SPDX-FileCopyrightText: © 2020-2023 Serpent OS Developers
+#
+# SPDX-License-Identifier: Zlib
+#
+name        : rust
+version     : 1.68.0
+release     : 1
+homepage    : https://git.kernel.org/cgit/linux/kernel/git/jaegeuk/f2fs-tools.git/about/
+upstreams   :
+    - https://static.rust-lang.org/dist/rustc-1.68.0-src.tar.xz : 8651245e8708f11d0f65ba9fdb394c4b9300d603d318045664b371729da9eac4
+summary     : A fast and secure system programming language
+description : |
+    Rust is a systems programming language that runs blazingly fast, prevents nearly all segfaults, and guarantees thread safety.
+license     :
+    - Apache-2.0
+    - MIT
+networking  : yes
+builddeps   :
+    - binary(FileCheck)
+    - binary(python3)
+    - pkgconfig(openssl)
+    - gcc-devel
+    - llvm-devel
+setup       : |
+    sed -e 's|PREFIX|%(prefix)|g' \
+        -e 's|LIBDIR|%(libdir)|g' %(pkgdir)/config.toml.in > config.toml
+build       : |
+    python3 x.py build --stage 1 -j %(jobs)
```

Comment:
Wouldn't it be better to use stage 2? https://rustc-dev-guide.rust-lang.org/building/bootstrapping.html#stage-2